doc/source/yt4differences.rst

```diff
@@ -159,24 +157,20 @@ It should be noted that the
     # generate an octree
     octree = ds.octree(left, right, n_ref=64)
 
+    ds.sph_smoothing_style = "scatter"
```

Comment:
It might be worth mentioning here that scatter is the default behavior (as an inline comment or something)